### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by samir-elhelw76**
We fixed bugs

### CODE REVIEW

**Review by briweinstein — APPROVED** [review]

**Review by lucas-m98 — APPROVED** [review]
Should be all the fixes I had.

File: packet_analysis/equation/generate_equation.py [hunk 1]
```diff
@@ -74,15 +74,15 @@ def get_segmented_average(conn: sqlite3.Connection, table: str, adjust_for_zero=
     return averaged_data, averaged_deviation
 
 
-# Seventh degree polynomial function
-def objective(x, a, b, c, d, e, f, g, h):
+# Fifth degree polynomial function
```
Comment (MartinLPetrauskas) [inline packet_analysis/equation/generate_equation.py:77]: Why did we change from to a 5th degree polynomial from 7th? Will this use less CPU when doing the complex calculations?

Comment (lucas-m98) [reply packet_analysis/equation/generate_equation.py:77]: Smaller degree, larger coefficients. The coefficient rounding was causing issues, and the degree of the polynomial was also more complex than necessary (Coefficients for the 6th and 7th expressions were minuscule)

Comment (MartinLPetrauskas) [reply packet_analysis/equation/generate_equation.py:77]: That makes sense. Thank you for clarifying.

**Review by MartinLPetrauskas — APPROVED** [review]
PR has been tested so that it compiles and runs properly. Had to remove the alert batching because it would cause the database to lock in unexpected ways

**Pull request merged**